app/views/users/show.html.erb

```diff
@@ -1,5 +1,18 @@
 <div>
-  <h1>Profile</h1>
-  <p>Username: <%= @user.name %></p>
-  <p>Email Address: <%= @user.email %></p>
+  <h1>Welcome, <%= @user.name %></h1>
+  <div><%= gravatar_tag @user.email %></div>
+
+  <h3>Questions</h3>
+  <% @user.questions.order(created_at: :desc)[0..9].each do |question| %>
```

Comment:
I'll chalk this up to being sick:

* But this sort of deep thinking certainly belongs in a controller
* check out `limit()` as an AR option to limit the number of results

So...in `users#show`

`@questions = @user.questions.order(created_at: :desc).limit(10)`

MO' EPIC.